tools/pipeline-generator/Azure.Sdk.Tools.PipelineGenerator/Conventions/IntegrationTestingPipelineConvention.cs

```diff
@@ -57,7 +57,7 @@ private PullRequestTrigger GetDefaultPrTrigger()
                 RequireCommentsForNonTeamMembersOnly = false,
                 IsCommentRequiredForPullRequest = true,
             };
-            newTrigger.BranchFilters.Add("+master");
```

Comment:
Is this all the references to master in the pipeline generator? It would be great to fix all of them as part of this change.

Reply:
Should be (I'll double check though). I actually parameterized the branch from the beginning. The pipeline generator templates themselves will need to be updated also, but that is a point in time change, and the way that the template is structured we can do it on a repo-by-repo basis.

Reply:
Found one more ... pushed a commit to cover it also.